train-1/Rotation.cpp

```diff
@@ -0,0 +1,12 @@
+class Solution {
+public:
+    void rotate(vector<int>& nums, int k) {
+        vector<int>::iterator ite;
+        int len=nums.size();
+        int cnt=k%len;
+        ite= nums.end()-cnt;
```

Comment:
You can just use `auto ite = nums.end() - cnt;`. `auto` is a good thing when you don't wanna write a long long type name.